databases/week2/Part 2 School database .SQL

```diff
@@ -0,0 +1,63 @@
+-- Create a new database containing the following tables:
+Class: with the columns: id, name, begins (date), ends (date)
+Student: with the columns: id, name, email, phone, class_id (foreign key)
+
+SET NAMES utf8mb4;
+
+
+
+
+CREATE TABLE `class1` (
+  
+`id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  
+`name` varchar(255) NOT NULL,
+  
+`begins` timestamp,
+  
+`ends` timestamp,
+  
+PRIMARY KEY (`id`)
+),
+ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+----------------------------------------------
+
+-- Student: with the columns: id, name, email, phone, class_id (foreign key)
+
+
+
+CREATE TABLE `student2` (
+
+`id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+
+`name` varchar(255) NOT NULL,
+
+`email` varchar(255) NOT NULL,
+
+`phone` varchar(255) NULL,
+
+`classid` int(10),
```

Comment:
It's good practice to use `_` in this case: `class_id`. This way you can easily see table name <-> column name